oarepo_requests/ui/theme/assets/semantic-ui/js/oarepo_requests_ui/common/utils/utils.js

```diff
@@ -39,24 +39,24 @@ export const getRequestStatusIcon = (requestStatus) => {
   }
 };
 
-export const getFeedMessage = (requestStatus, creatorLabel) => {
-  switch (requestStatus?.toLowerCase()) {
+export const getFeedMessage = (eventStatus) => {
```

Comment:
@mirekys Ahoj, the problem is that you cannot really use interpolated string as a key, because this key will never end up in the .po file as the make-translation (catalog compiler part I guess) has no idea what this key is supposed to be actually. So if you create such a key and call make-translations, it does not go to the .po file. If there is some trick to it, please let me know.

Reply:
You're right :)